tests/integration/ha/test_backups.py

```diff
@@ -232,19 +250,7 @@ async def test_backup_cluster(
     logger.info(f"create-backup output: {action}")
 
     assert action.status == "completed"
-
-    list_backups = await run_action(ops_test, leader_id, "list-backups", params={"output": "json"})
-    logger.info(f"list-backups output: {list_backups}")
-
-    # Expected format:
-    # namespace(status='completed', response={'return-code': 0, 'backups': '{"1": ...}'})
-    backups = json.loads(list_backups.response["backups"])
-    assert list_backups.status == "completed"
-    assert len(backups.keys()) == int(action.response["backup-id"])
-    assert backups[action.response["backup-id"]]["state"] == "SUCCESS"
-
-    # continuous writes checks
-    await assert_continuous_writes_consistency(ops_test, c_writes, app)
```

Comment:
Why don't we need to check for this anymore?

Reply:
I do, but if I reuse in the disaster recovery scenarios, then this test won't really work b/c effectively the writes will break for a certain time.

Reply:
I am returning these checks, but taking another route to reuse certain tests.